src/mpc-hc/mpc-hc.rc

```diff
@@ -1510,9 +1543,9 @@ BEGIN
         MENUITEM "&Filters",                    ID_FILTERS
         MENUITEM "S&haders",                    ID_SHADERS
         MENUITEM SEPARATOR
-        MENUITEM "&Audio",                      ID_AUDIOS
-        MENUITEM "Su&btitles",                  ID_SUBTITLES
-        MENUITEM "&Video Stream",               ID_VIDEO_STREAMS
+        MENUITEM "&Video Streams",              ID_VIDEO_STREAMS
+        MENUITEM "&Audio Streams",              ID_AUDIOS
+        MENUITEM "Su&btitles Streams",          ID_SUBTITLES

```

Comment:
Sure, i wanted to be consistent in naming (since all are streams), and I think the order is better too.